docs/adrs/0001-different-states.md

```diff
@@ -0,0 +1,23 @@
+# Returning different states in the API
+
+## Context 
+
+Currently, this stub API works quite basically; doing a very basic level of validation and responding with mostly static responses.
+
+As we develop One Login Home, there is a need to have a more flexibility in the API responses. For example, with MFA method management, we'd like to demonstrate users who have different combinations of MFA factors enabled, we can't do this with static responses.
+
+## Option 1: Systems Manager Parameter Store
+
+We could use the [systems manager parameter store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) to store a parameter that corresponds to a scenario. And then return the response we want.
+
+This would work in a similar way to how we use the secrets manager, in the lambda, we would read what the parameter is. The parameter could be changed either through the AWS console, or through an API. We could create a simple interface in the stub api to allow people to change the current scenario.
```

Comment:
```suggestion
This would work in a similar way to how we use the secrets manager currently.  In the lambda we would read read the parameter from secrets manager and use it to determine the response. The parameter could be changed either through the AWS console, or through an API. We could create a simple interface in the stub api to allow people to change the current scenario.
```